services/src/main/scala/cromwell/services/auth/GithubAuthVendingSupport.scala

```diff
@@ -13,16 +18,16 @@ import cromwell.services.auth.GithubAuthVending.{
   NoGithubAuthResponse
 }
 
+import net.ceedubs.ficus.Ficus._
 import scala.concurrent.{ExecutionContext, Future}
 
 trait GithubAuthVendingSupport extends AskSupport with StrictLogging {
 
   def serviceRegistryActor: ActorRef
 
-  implicit val timeout: Timeout
```

Comment:
I don't necessarily have an objection, just curious - why is the timeout implicit rather than overridden by classes that extend this trait?

Reply:
The reason I removed it from here and put it as implicit parameter for each method was to address [this comment](https://github.com/broadinstitute/cromwell/pull/7373#discussion_r1501254668).

But in regards to your specific question, I am not really sure. I have seen this pattern for timeout being implicit in other actor classes and my understanding was making them implicit means it could be extracted from the context indirectly (as long as it is in the context) when needed instead of each class having to supply it. But my understanding is basic. Maybe @cjllanwarne knows more?